Backend/controllers/events.js

```diff
@@ -1,5 +1,6 @@
 import dotenv from "dotenv";
 import Event from "../models/events.js";
+const createCsvWriter = require("csv-writer").createObjectCsvWriter;
```

Comment:
Ensure consistency in our codebase by sticking to ES6 module imports and exports syntax throughout our application. This helps maintain code clarity and aligns with modern JavaScript best practices.